cl/opinion_page/forms.py

```diff
@@ -237,6 +258,70 @@ class CourtUploadForm(forms.Form):
             }
         ),
     )
+
+    disposition = forms.CharField(
+        label="Disposition",
+        required=False,
+        widget=forms.Textarea(
+            attrs={
+                "class": "form-control",
+            }
+        ),
+    )
+
+    summary = forms.CharField(
+        label="Summary",
+        required=False,
+        widget=forms.Textarea(attrs={"class": "form-control"}),
+    )
+
+    # TODO this could be a select field with hardcoded options for a specific court
+    lower_court_str = forms.CharField(
+        label="Lower Court",
+        required=False,
+        widget=forms.TextInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "Lower court name",
+                "autocomplete": "off",
+            }
+        ),
+    )
+
+    lower_court_docket_number = forms.CharField(
+        label="Lower Court docket number",
+        required=False,
+        widget=forms.TextInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "Lower court docket number",
+                "autocomplete": "off",
+            }
+        ),
+    )
+
+    download_url = forms.URLField(
+        validators=[URLValidator()],
+        label="Download Url",
+        required=False,
+        widget=forms.TextInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "The URL where the document is originally located",
+                "autocomplete": "off",
+            }
+        ),
+    )
```

Comment:
Is this the correct description?  since we aren't using a scraper and the upload is being done manually, perhaps we dont bother?  @mlissner

Reply:
The description seems right to me. It seems like a nice thing to have the URL if we can.